source/blog/2018-07-10-Skydive-With-oVirt.html.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Skydive With oVirt 
+author: mburman
+tags: oVirt, oVirt 4.2, open source virtualization,network
+date: 2018-07-10 09:01:00 UTC
+---
+
+[Skydive network](http://skydive.network/) is an open source real-time network topology and protocols analyzer providing a comprehensive way of understanding what is happening in your network infrastructure.
+The common use cases will be, troubleshooting, monitoring, SDN integration and much more.
+It has features such as:
+* Topology capturing - Captures network topology, interface, bridge and more
+* Flow capture - Distributed probe, L2-L4 classifier, GRE, VXLAN, GENEVE, MPLS/GRE, MPLS/UDP tunnelling support
+* Extendable - Support for external SDN Controllers or container based infrastructure, OpenStack. Supports extensions through API
+
+## Benefit to oVirt users
+Skydive allows oVirt administrators to see the network configuration and topology of their oVirt cluster.
+Administrators can capture traffic from VM1 to VM2 or monitor the traffic between VMs or hosts. 
+Skydive can generate traffic between 2 running VMs on different hosts and then analyze.
+Administrators can create alerts in Skydive UI to notify when traffic is disconnected or down.
+
```

Comment:
Please insert
`READMORE`
here. Else the installation steps will be shown in the blog's front page.
See https://middlemanapp.com/basics/blogging/#article-summaries for explanation or other oVirt blog posts for examples.
You can have a local preview by
`sudo ./setup.sh` and `sudo ./run-server.sh`, see README.md and CONTRIBUTING.md.

Reply:
sorry, should be just READMORE without backticks